teammapper-frontend/mmp/src/map/map.ts

```diff
@@ -85,37 +86,100 @@ export default class MmpMap {
      * @return {MmpInstance} mmpInstance
      */
     private createMmpInstance(): MmpInstance {
```

Comment:
we have talked about this, to revert this change and leave it as it was? What did change your opinion that its still this way?